src/scripts/main.js

```diff
@@ -1,5 +1,38 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
```

Comment:
Similar to the previous comment, it would be a good idea to check if the wall element exists.